### PR TITLE
docs/customizing-install: update for `iso/pxe customize`

### PR DESCRIPTION
**Pull request by bgilbert**
Deemphasize customizing via kernel arguments and manual Ignition embedding, since `iso/pxe customize` has better UX.

Drop documentation of manual pre/post hooks, since it's very clunky and `iso/pxe customize` can handle this.

### CODE REVIEW

File: docs/customizing-install.md [hunk 1]
```diff
@@ -8,10 +8,77 @@ nav_order: 5
...
+  `--live-karg-replace`, `--live-karg-delete`).  There are no `--live-karg`
+  options for the PXE image; modify the PXE boot configuration instead.
+- Running scripts before or after installation (`--pre-install`,
+  `--post-install`).  For example, a pre-install script might run a
+  container that performs hardware validation, or a post-install script
+  might use IPMI to configure the machine to boot from the local disk
+  instead of the network.  Pre-install scripts can change the options
+  processed by coreos-installer, including the choice of destination device,
+  by writing an installer config file to `/etc/coreos/installer.d` (see
+  below).
```
Comment (jlebon) [inline docs/customizing-install.md:71]: In a follow-up, it'd probably be worth fleshing this out with some examples. E.g. a script which uses the MAC address to auto-generate an NM keyfile with static IP to copy in? Or automatically select the destination device based on disk model?

**Review by jlebon — APPROVED** [review]
Nice!

File: docs/customizing-install.md (hunk 1 above)
Comment (bgilbert) [reply docs/customizing-install.md:71]: https://github.com/coreos/coreos-installer/issues/784

**Comment by bgilbert** [comment]
Moved some additional language here that was dropped from https://github.com/coreos/fedora-coreos-docs/pull/355.

**Pull request merged**